scripts/validate-cloudtrail-logs.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+##? Usage: validate-cloudtrail-logs.sh [options] ACCOUNT START_FROM [UP_TO]
+##?
+##? Options:
+##?   --config=FILE  Path to file containing details of trails (default: $DM_CREDENTIALS_REPO/jenkins-vars/cloudtrail.yaml)
+##?   -v, --verbose
+##?   -n, --dry-run
+##?   -h, --help
+
+set -eo pipefail
+
+help=$(grep "^##?" "$0" | cut -c 5-)
```

Comment:
Can't we use the pattern we've used elsewhere?

https://github.com/alphagov/digitalmarketplace-aws/blob/master/scripts/map-route.sh#L4-L3

Reply:
We could, but I found using docopts a lot easier, I think it has a lot of advantages over the alternatives, especially given that we use docopt in all our python scripts.